app/models/subscriber.rb

```diff
@@ -0,0 +1,32 @@
+class Subscriber
```

Comment:
just noticed. Is `models` really the appropriate directory for this? Why not `services` or something similar?